src/components/PostSection/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import ImageWithSideTitle from '../ImageWithSideTitle/index';
+import { ReusableDiv } from '../DivWithCenterdChildren/index';
+import PopUp from '../OptionsBtn/index';
+import { ShareBtn } from '../ShareBtn';
+
+const PostSection = ({
+  title,
+  subTitle,
+  imgURL,
+  paragraph,
+  bgImageOne,
+  bgImageTwo,
+  divHeight,
+  smallRound,
+  bgColor,
+  pickIcon,
+  searchIcon,
+  saveIcon,
+  options,
+  url
```

Comment:
You don't have to add all these props.
it's not a reusable component, it's a section.
and make the prop name more readable and describes what it used in
the props you need here is just **( userImage, userName, postTime, postCaption, rightImage, leftImage, votesNumber )** and we will add more to it later, but don't handle the styles by the props.

Reply:
It's just icons to show every thing as it is in design. Also options prop is coming from the PopUp component a component you made.